packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/angular/localsignout.md

```diff
@@ -0,0 +1,18 @@
+Call the `logout` method on the [OktaAuthService](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-angular#oktaauthlogouturi). 
```

Comment:
Call the `logout` method on [OktaAuthService](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-angular#oktaauthlogouturi).